pages/index.tsx

```diff
@@ -22,9 +22,11 @@ const MainPage = ({ stories }: MainPageProps) => {
         <StoryFeed stories={stories} />
       </Box>
       <FloatingRibbon>
-        <NextLink href="/new">
+        <NextLink href="/new" passHref>
           <Link>
-            <Button>Add Your Story</Button>
+            <Button marginTop="5px" marginBottom="5px">
```

Comment:
Can be simplified using Chakra's `my` https://chakra-ui.com/docs/features/style-props

```suggestion
            <Button my={"5px"}>
```

Reply:
Thanks, didn't know that!